cloud/team/team.go

```diff
@@ -0,0 +1,708 @@
+package team
+
+import (
+	httpContext "context"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
```

Comment:
maybe use `errors` instead, since `pkg/errors` is archived project

Reply:
done